implementations/rust/ockam/ockam_api/src/nodes/service/services.rs

```diff
@@ -354,6 +355,21 @@ impl NodeManager {
             )
         };
 
+        // override default policy to allow incoming packets from the project
+        if let Some(project_name) = &project_name {
+            let (_addr, identity_identifier) = self.resolve_project(project_name)?;
+            self.policies
+                .set_policy(
+                    &resources::INLET,
+                    &actions::HANDLE_MESSAGE,
+                    &eq([
+                        ident("subject.identity"),
```

Comment:
What happens if this attribute is missing?
We had some issue with missing attributes evaluation and I'm not sure what implementation ended up being like.

Reply:
Being the identity identifier itself can't be missing from a secure channel context, the check would fail before being evaluated.

https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_abac/src/policy.rs#L93

Reply:
I must be missing the place where we enforce that the message comes from a secure channel.
There was an issue with attributes missing vs attribute values mismatch in the `eq` rule in the rust implementation, which might cause some errors.

Reply:
As @davide-baldo pointed out, on line 93 we check if some identity info is associated to the message. If the message does not come from a secure channel, then this is not the case and we don't evaluate the policy